UIforETW/ChildProcess.cpp

```diff
@@ -140,18 +143,20 @@ bool ChildProcess::Run(bool showCommand, std::wstring args)
 	DWORD flags = CREATE_NO_WINDOW;
 	// Wacky CreateProcess rules say args has to be writable!
 	std::vector<wchar_t> argsCopy(args.size() + 1);
-	wcscpy_s(&argsCopy[0], argsCopy.size(), args.c_str());
-	BOOL success = CreateProcess(exePath_.c_str(), &argsCopy[0], NULL, NULL,
+	const int res = wcscpy_s(&argsCopy[0], argsCopy.size(), args.c_str());

```

Comment:
> wcscpy_s already has internal error checking so this adds no value.

The VS2013 headers are messy enough (`#define _FUNC_NAME wcscpy_s` - *really, are you kidding me???!?*) that I'll take your word for it.

> It's sort of like going: "if (ptr) delete ptr;" - since 'delete' promises to do the 'if' check it is just bloat if user-code does it as well.

Ahh, I hit your pet-peeve :)